kornia/constants.py

```diff
@@ -1,6 +1,9 @@
 from typing import Union, TypeVar
 from enum import Enum
 
+import torch
+
+PI = torch.tensor(3.14159265358979323846)
```

Comment:
leave it in undercase, was inspired by numpy here: https://docs.scipy.org/doc/numpy/reference/constants.html#numpy.pi